### PR TITLE
Adding Configuration for appveyor ci

### PR DESCRIPTION
**Pull request by ishubham-k**
A while I was having a lot of trouble in running ```sbt test``` on windows machine. Reasons are still unknown why that was happening(may be jsdom or may be wrong version of npm). So I thought adding a appveyor ci config to give some love to windows users :)

### CODE REVIEW

**Comment by ochrons** [comment]
This seems to be congfigured specifically for your system, with all those absolute paths etc. Probably not very useful for users with different paths?

**Comment by ishubham-k** [comment]
I think that should not be a problem. Not sure what absolute paths you are referring to.

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+version: 1.0.{build}
+
+environment:
+  PATH: C:\sbt\bin;%PATH%
```
Comment (ochrons) [inline appveyor.yml:4]: These absolute paths won't work for other users.

**Comment by ochrons** [comment]
For example SBT path and JAVA_HOME.

Besides, this is quite a bit outside the scope of the SPA tutorial anyway :)

File: appveyor.yml (hunk 1 above)
Comment (ishubham-k) [reply appveyor.yml:4]: The directory for extracting the zip is specified in appveyor.ps1 script line number 13. This absolute path is just a reference to that.

**Comment by ishubham-k** [comment]
Agreed. Closing this one. Thanks.

**Pull request closed without merging**